tests/sanity/ignore-2.9.txt

```diff
@@ -62,7 +62,6 @@ plugins/modules/vyos_logging.py validate-modules:doc-missing-type
 plugins/modules/vyos_logging.py validate-modules:missing-suboption-docs
 plugins/modules/vyos_logging.py validate-modules:parameter-type-not-in-doc
 plugins/modules/vyos_logging.py validate-modules:undocumented-parameter
-plugins/modules/vyos_ospfv2.py validate-modules:invalid-documentation
```

Comment:
since we are doing this, we should go though and fix all invalid-documentation ignores. As it is possible there is another ansible-doc failure coming.